hebrew.html

```diff
@@ -296,15 +296,17 @@ <h2>שמרו איתנו על קשר</h2>
   </div>
 
   <div class="footer">
-    <div class="content">
-      <p>
-        כל הזכויות שמורות
-        © 2021
-      </p>
-      <p>
-        <a href="http://www.freepik.com">גרפיקה של Freepik / macrovector</a> <a href="https://storyset.com/people">/
-          Freepik Storyset</a>
-      </p>
+    <div class="content">  
+        <img class="footerLogo" src="assets/whiteLogo.svg"/>       
+        <p class="credits">
+          Qint 2021. כל הזכויות שמורות          
+        </p>       
+        <navbar>
+          <a>אבטחה</a><span class=dividor>|</span>
```

Comment:
You're not styling the `dividor` class anywhere so why declare it?

Reply:
Oh right in the end I just used the span tag, it was initially a div with the class.